rosbag2_storage/include/rosbag2_storage/bag_metadata.hpp

```diff
@@ -33,14 +33,16 @@ struct TopicInformation
 
 struct BagMetadata
 {
-  int version = 2;  // upgrade this number when changing the content of the struct
+  int version = 3;  // Upgrade this number when changing the content of the struct
```

Comment:
Let's split modifying the metadata into a separate PR

Reply:
What if the metadata is required for functionality, i.e., pushing this PR without modifying metadata is non-functional code?